src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -44,35 +44,37 @@ const initialValuesOrganizationForm: OrganizationForm = {
 };
 
 interface SelectInstitutionFormProps {
-  addAffiliation: (id: string) => void;
-  onClose?: () => void;
-  suggestedInstitutions: string[];
+  saveAffiliation: (id: string) => void;
+  onCancel?: () => void;
+  suggestedInstitutions?: string[];
+  initialValues?: OrganizationForm;
```

Comment:
Takes in optional initial values so that in edit-mode the input-fields are pre-filled